torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -1046,7 +1046,8 @@ class TensorExprFuser {
     }
 
     if (node->kind() == aten::conv2d) {
-      if (!tensorexpr::conv2dIsSupportedJit(node)) {
+      if (!tensorexpr::conv2dIsSupportedJit(node) &&
+          !tensorexpr::mkldnnPrepackedConvIsSupportedJit(node)) {
```

Comment:
Should the condition be `||` here?

Reply:
Here we want to let NNC selects `conv2d` into the fusion group if either `conv2dIsSupportedJit` (depthwise, 3x3 kernel) or `mkldnnPrepackedConvIsSupportedJit` returns true.

`conv2dIsSupportedJit` only returns true for depthwise conv2d of 3x3 kernel. For other shapes that mkldnn prepacked conv2d can provide performance boost, we still want these `conv2d` OPs to be rewritten.